### ISSUE TITLE
Accessibility issues

### ISSUE THREAD

**Issue by with-heart** [narendrasss/NotANumber#72]
Looooooove what you're doing with your blog. It's one of my favorite things!

While looking through it and poking around, I noticed some UX issues that I wanted to report:

# Pop quiz in dark mode isn't readable

(hovered button on left, selected button on right, success text underneath)

<img width="622" alt="image" src="https://user-images.githubusercontent.com/1954752/210273572-779adf86-6c04-4a70-be39-6e7eed45eccb.png">

Looks great in light mode though: 
<img width="626" alt="image" src="https://user-images.githubusercontent.com/1954752/210273408-ba80f1b5-23c6-4aeb-93ee-b22f0e6d88e6.png">

# Lacking focus indicators for keyboard navigation

Here's a screencapture of me tabbing my way through part of the page:

![Kapture 2023-01-02 at 14 58 34](https://user-images.githubusercontent.com/1954752/210273953-c69cd71c-4735-43d6-b284-ec2ca78bceb0.gif)

Aside from links, buttons from the code previews, and accordion headers, there's no focus indicator to tell me what I currently have selected. Makes it almost impossible to navigate effectively with a keyboard!
